src/modules/auth/AccessControlScreen.js

```diff
@@ -65,32 +65,40 @@ function AccessControlCard(props) {
 			</CardHeader>
 
 			<CardBody>
-				{App.Services.TenantService &&
-					<React.Fragment>
+				{ userinfo ? (
+					<>
+						{App.Services.TenantService &&
+							<React.Fragment>
+								<Row>
+									<Col>
+										<h5>{t('AccessControlScreen|Tenant')}</h5>
+									</Col>
+									<Col>
+										<p style={{marginBottom: "5px"}}>{currentTenant}</p>
+									</Col>
+								</Row>
+								<hr/>
+							</React.Fragment>
+						}
 						<Row>
 							<Col>
-								<h5>{t('AccessControlScreen|Tenant')}</h5>
+								<h5>{t('AccessControlScreen|Roles')}</h5>
 							</Col>
+							<ItemToRender userinfo={userinfo} item='roles' />
+						</Row>
+						<hr/>
+						<Row>
 							<Col>
-								<p style={{marginBottom: "5px"}}>{currentTenant}</p>
+								<h5>{t('AccessControlScreen|Resources')}</h5>
 							</Col>
+							<ItemToRender userinfo={userinfo} item='resources' />
 						</Row>
-						<hr/>
-					</React.Fragment>
-				}
-				<Row>
-					<Col>
-						<h5>{t('AccessControlScreen|Roles')}</h5>
-					</Col>
-					<ItemToRender userinfo={userinfo} item='roles' />
-				</Row>
-				<hr/>
-				<Row>
-					<Col>
-						<h5>{t('AccessControlScreen|Resources')}</h5>
-					</Col>
-					<ItemToRender userinfo={userinfo} item='resources' />
-				</Row>
+					</>
+					) : (
+					<div className="text-center">
+						<h5>{t("AccessControlScreen|UserInfo doesn't exist.")}</h5>
```

Comment:
The message is "weird", I suggest:
"The user information is invalid, you session is likely expired."